app/pages/event1/BenefitInKindBriefDescriptionPage.scala

```diff
@@ -29,4 +30,11 @@ case object BenefitInKindBriefDescriptionPage extends QuestionPage[String] {
 
   override def route(waypoints: Waypoints): Call =
     routes.BenefitInKindBriefDescriptionController.onPageLoad(waypoints)
+
+  override protected def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
+    answers.get(BenefitInKindBriefDescriptionPage) match {
```

Comment:
afaik, form is optional and we don't need to check.
PaymentValueAndDatePage is only option

Reply:
Resolved in next commit